src/Appwrite/Platform/Modules/Databases/Http/Databases/Collections/Attributes/Boolean/Create.php

```diff
@@ -51,7 +51,10 @@ public function __construct()
                         code: SwooleResponse::STATUS_CODE_ACCEPTED,
                         model: $this->getResponseModel(),
                     )
-                ]
+                ],
+                deprecated: [
+                    'replaceWith' => 'createBooleanColumn',
```

Comment:
this `'replaceWith' => 'createBooleanColumn',` - infers that the method is available on the same namspace, i.e. `Databases` which is not the case.